### ISSUE TITLE
Is it possible to close a document on the server

### ISSUE THREAD

**Issue by vkalpias** [share/ShareJS#387]
I am trying to close and delete a document from the server when all clients disconnect. Is this possible somehow?

I've tried manually deleting the document from the database and all its related ops (using livedb with mongo). The problem is that after its deleted manually, when it gets requested again we recreate it with the same id. This causes issues as probably the document is still open on the server with a different version..

Basically I want to remove the document from memory and from the database - can this be done?
